platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/StandardThreadFactoryBuilder.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class StandardThreadFactoryBuilder implements ThreadFactory {
+    static final AtomicLong count = new AtomicLong(0);
+
+    public static ThreadFactory newThreadFactory(String threadName) {
+        return newThreadFactory(threadName, true, Thread.NORM_PRIORITY);
+    }
+
+    public static ThreadFactory newThreadFactory(String threadName, boolean isDaemon,
+            int priority) {
+        if (threadName == null || threadName.isEmpty()) {
+            threadName = "thread";
+        }
+        return build(threadName, isDaemon, priority);
+    }
+
+    private static ThreadFactory build(String threadName, boolean isDaemon, int priority) {
+        return r -> {
```

Comment:
✏️ I'm not sure if I misread this earlier and it was already correct or if the `AtomicLong` was incorrectly inside the factory method. 

Regardless, the right way to do this is to get rid of the global and create your new `AtomicLong` on line 35, right before your return statement. You can explicitly mark as final if you want, or just let Java treat it as effectively final for you.